```running tests
check flags: -j64 -l64 SHELL=/nix/store/q4116dsmdsab1wgnzdknlpfmxsx897cn-bash-4.4-p23/bin/bash VERBOSE=y test
make: *** No rule to make target 'test'.  Stop.
builder for '/nix/store/vg4d0qp130mx6gfxj1599vbff58iksyf-range-v3-0.10.0.drv' failed with exit code 2
error: build of '/nix/store/vg4d0qp130mx6gfxj1599vbff58iksyf-range-v3-0.10.0.drv' failed```
Okay, doCheck should be disabled too

--- a/pkgs/development/libraries/range-v3/default.nix
+++ b/pkgs/development/libraries/range-v3/default.nix
@@ -13,7 +13,11 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake ];
 
-  doCheck = true;
+  # Building the tests currently fails on AArch64 due to internal compiler
+  # errors (with GCC 9.2):
+  cmakeFlags = stdenv.lib.optional stdenv.isAarch64 "-DRANGE_V3_TESTS=OFF";
+
+  doCheck = !stdenv.isAarch64;
   checkTarget = "test";
 
   enableParallelBuilding = true;

